bucket/windbg.json

```diff
@@ -0,0 +1,62 @@
+{
+    "homepage": "https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/",
+    "description": "The official Windows debugger",
+    "version": "10.0.19041.0",
+    "url": "https://go.microsoft.com/fwlink/p/?linkid=2120843#/temp/sdk-setup.exe",
+    "hash": "d53f651370f87484b78622e30dfb1a41920b501e4041035771c0d785561f47d5",
+    "installer": {
+        "script": [
+            "Expand-DarkArchive \"$dir\\sdk-setup.exe\" \"$dir\\temp\"",
+            "$setupSettings = (Select-Xml -Path \"$dir\\temp\\UX\\UserExperienceManifest.xml\" -XPath '/').Node.UserExperienceManifest.Settings",
+            "$version = $setupSettings.ProductName.Split(' ')[-1]",
+            "Write-Output \"Actual version is $version\"",
+            "$downloadRootPointer = $setupSettings.SourceResolution.DownloadRoot",
+            "$downloadRoot = (Invoke-WebRequest -Method Head -Uri $downloadRootPointer -MaximumRedirection 0 -ErrorAction Ignore).BaseResponse.Headers['Location']",
+            "$manifestPayloads = (Select-Xml -Path \"$dir\\temp\\UX\\manifest.xml\" -XPath '/').Node.BurnManifest.Payload",
+            "$windbgPayload = $manifestPayloads | Where-Object { $_.FilePath.Contains(\"X$($architecture.Replace('bit', '')) Debuggers And Tools\") }",
+            "$windbgDownloadPath = $windbgPayload.SourcePath",
+            "$windbgHash = $windbgPayload.Hash",
+            "dl_with_cache $app $version ($downloadRoot + $windbgDownloadPath) \"$dir\\temp\\windbg-setup.msi\"",
```

Comment:
Never do this in official buckets. Downloading some random files without hash in bunch of lines of code is dangerous.

If this will not be fixed it will be closed.

Reply:
How normally?
There is no way to get the download url with autoupdate, because the actual url is inside another WiX installer as I've explained.
How is that different than using checkver+autoupdate to get the url from a webpage?

What's your suggestion? I even used `dl_with_cache` to make it have an indicative progress bar.

Reply:
Using `url` property.

Then use it in own bucket.

Allowing manifests with such practice will lead into junk and dangerous software to be added into official buckets.

Reply:
@Ash258 

This is an *official* Microsoft product, which has a complex installation process.

This is the exact opposite of junk.

The problem here is that scoop can't handle this installer indirection.

If git had a complex installer which downloads another installer, would you also not include it in the official buckets?

Please reconsider. This is a very useful app for many people, created *by Microsoft* especially *for Windows*.
scoop is supposed to be a Windows app installer, it's like not having .NET in scoop.